tests/plugins/uploadwidget/manual/destroyeditor.html

```diff
@@ -0,0 +1,21 @@
+<div id="editor"></div>
+<button id="destroyer">Destroy</button>
+
+<script>
+	CKEDITOR.replace( 'editor', {
+		imageUploadUrl: 'http://sub.ckeditor.dev/',
+		on: {
+				instanceReady: function( evt ) {
+					var editor = evt.editor;
+
+					document.getElementById( 'destroyer' ).addEventListener( 'click', function() {
+						editor.destroy();
+					} );
+
+					if ( !CKEDITOR.fileTools.isFileUploadSupported || bender.tools.env.mobile ) {
```

Comment:
This should be moved to the beginning and return early with ignoring. Such move would prevent mentioned errors in IE 8.